src/lib/transaction_logic/test/transaction_logic.ml

```diff
@@ -505,6 +505,35 @@ let%test_module "Test transaction logic." =
                       ] ) ) )
             (run_zkapp_cmd ~fee_payer ~fee ~accounts txns) )
 
+    let%test_unit "Update_verification_key can not be set to be proof or \
+                   impossible" =
+      Quickcheck.test ~trials
```

Comment:
Can we simply test the two concrete cases- one for `Proof` and one for `Impossible` instead of doing 100 random trials?